apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/VedtaksbrevService.kt

```diff
@@ -129,9 +130,12 @@ class VedtaksbrevService(
         val brevkodePar = brevDataMapper.brevKode(generellBrevData, brev.prosessType)
 
         val brevData =
-            when (migrering) {
-                null -> opprettBrevData(brev, generellBrevData, brukerTokenInfo, brevkodePar)
-                else -> migreringBrevDataService.opprettMigreringBrevdata(generellBrevData, migrering, brukerTokenInfo)
+            when (generellBrevData.systemkilde == Vedtaksloesning.PESYS) {
+                false -> opprettBrevData(brev, generellBrevData, brukerTokenInfo, brevkodePar)
+                true ->
+                    OmregnetBPNyttRegelverkFerdig(
```

Comment:
Dette skurrar, for automatisk migrering har vi vel ikkje lagra innhald?

Reply:
Hm. Det blir vel lagret under opprett-steget? Forskjellen er vel bare at til nå så brukes ikke det lagre innholdet til å generere pdfen.
Noe vi må gjøre for å kunne redigere innholdet for manuell migrering.

Reply:
Ja, det er jo eit poeng, så da bør det vel fungere.